crates/sui-core/src/authority/authority_notify_read.rs

```diff
@@ -49,6 +49,12 @@ pub(crate) struct NotifyRead<K, V> {
     count_pending: AtomicUsize,
 }
 
+impl<K: Eq + Hash + Clone, V: Clone> Default for NotifyRead<K, V> {
```

Comment:
Any reason we cannot use derive?

Reply:
removed (added because of lint, but the lint is now gone, somehow ...)

Reply:
removed (added because of lint, but the lint is now gone, somehow ...)

Btw derive does not work because the default implementation (if we ever need one) should work as `fn new` where we do a bunch of things.